### PR TITLE
Added instance URL support for inventory count  (#1wuu9yg)

### PR DESCRIPTION
**Pull request by bashu22tiwari**
Screenshot from 2021-12-20 13-18-07

### CODE REVIEW

File: src/store/modules/user/actions.ts [hunk 1]
```diff
@@ -78,8 +78,15 @@ const actions: ActionTree<UserState, RootState> = {
     },
 
   // update current facility information
-  async setFacility ({ commit }, payload) {
+   async setFacility ({ commit }, payload) {
```
Comment (ymaheshwari1) [inline src/store/modules/user/actions.ts:81]: Improve Indentation

File: src/store/modules/user/actions.ts [hunk 2]
```diff
@@ -78,8 +78,15 @@ const actions: ActionTree<UserState, RootState> = {
...
     commit(types.USER_CURRENT_FACILITY_UPDATED, payload.facility);
+  },
+
+  /**
+   * Set User Instance Url
+   */
+   setUserInstanceUrl ({ state, commit }, payload){
```
Comment (ymaheshwari1) [inline src/store/modules/user/actions.ts:88]: Improve Indentation

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: src/store/modules/user/actions.ts (hunk 1 above)
Comment (bashu22tiwari) [reply src/store/modules/user/actions.ts:81]: Improved Indentation.
Thank You sir for reviewing my code.

File: src/store/modules/user/actions.ts (hunk 2 above)
Comment (bashu22tiwari) [reply src/store/modules/user/actions.ts:88]: Improved Indentation.
Thank You sir for reviewing my code.

**Review by adityasharma7 — APPROVED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

**Pull request merged**